src/agent/debuglet.ts

```diff
@@ -593,8 +651,14 @@ export class Debuglet extends EventEmitter {
                   that.logger.info(formatBreakpoints(
                       'Active Breakpoints: ', that.activeBreakpointMap));
                 }
-                that.scheduleBreakpointFetch_(
-                    that.config.breakpointUpdateIntervalSec);
+                if (onlyOnce) {
+                  that.breakpointFetchedTimestamp = Date.now();
```

Comment:
Would you want to set the fetched timestamp regardless of the value of `onlyOnce`?

Reply:
Yes. thanks for spotting this out. I think it makes sense to update the timestamp every time updateBreakpoint is called.